Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md

```diff
@@ -0,0 +1,148 @@
+- **Settings model** - Add a Settings section to your Block based on the given ElementType. When picked you can open the ElementType or choose to remove the settings section again.
+
+**Catalogue apperance**
+These properties refers to how the Block is presented in the Block catelogue, when editors choose which Blocks to use for their content.
+
+- **Background color** - Define a background color to be displayed beneath the icon or thumbnail.
+- **Icon color** - Change the color of the ElementType icon.
+- **Thumbnail** - Pick an image or SVG file to replace the icon of this Block in the catalogue.
+
+**Advanced**
+These properties are relevant when you work with custom views.
+
+- **Force hide content editor** - If you made a custom view that enables you to edit the content part of a block and you are using default editing mode(not inline) you might want to hide the content-editor from the block editor overlay.
+
+
+## Editing Blocks
+
+When viewing a **Block List** editor for the first time, you'll be presented with the option to Add content
+
+![Block List - Add Content](images/BlockListEditor_AddContent.png)
+
+Clicking the Add content button brings up the Block Catalogue
+
+![Block List - Simple setup](images/BlockListEditor_BlockPicker_simplesetup.jpg)
+
+The Block Catalogue looks different depending on the amount of avaiable Blocks and their catalogue apperance.
+
+![Block List - example setup from Umbraco.com](images/BlockListEditor_BlockPicker.jpg)
+
+Click the Block type you wish to create and a new Block will appear in the list.
+
+Depending on wether your Block List Editor is setup to use default or inline editing mode.
+In default mode you will enter the editing overlay of that Block:
+
+![Block List - simple overlay editing](images/BlockListEditor_EditingOverlay.jpg)
+
+In inline editing mode the new Blocks inline editor will expand:
+
+![Block List - simple inline editing](images/BlockListEditor_InlineEditing.jpg)
+
+
+More Blocks can be added to the list by clicking the Add content button or using the inline Add content button that appears on hover between or above existing Blocks.
+
+![Block List - Add Content](images/BlockListEditor_AddContentInline.jpg)
+
+To reorder the Blocks, click and drag a Block up or down to place in the decirede order.
+
+To delete a Block click the trash-bin icon appearing on hover.
+
+## Rendering Block List Content
+
+Rendering the stored value of your **Block List** property can be done in two ways.
+
+### 1. Default rendering
+Use the build-in rendering machinsm for rendering blocks via a Partial View for each block.
+
+To make this work you will need to create a Partial View for each block, named by the alias of the ElementType thats begin used as Content Model.
```

Comment:
I'm not sure I understand this sentence.
Does it mean that the name of the Partial View needs to be "Content Model" + the alias of the ElementType (block)?
I think it would make sense to add an example of how to name the Partial View.

Reply:
What I've figured from fiddling is that they should have the same name as the alias and then be put in the `Views/Partials/BlockList/Components/` folder.

Reply:
This has been updated, let me know if its better or if you have other improvements for this?